src/util/sharing_map.h

```diff
@@ -197,6 +202,12 @@ class sharing_mapt
   typedef std::vector<delta_view_itemt> delta_viewt;
 
   void get_view(viewt &view) const;
+  viewt get_view() const
+  {
+    viewt result;
+    get_view(result);
+    return result;
```

Comment:
Do we trust the compiler to optimise this appropriately and not end up with an extra copy here?